examples/flows/standard/conditional-flow-for-if-else/flow.dag.yaml

```diff
@@ -0,0 +1,47 @@
+$schema: https://azuremlschemas.azureedge.net/promptflow/latest/Flow.schema.json
+inputs:
+  question:
+    type: string
+    default: What is Prompt flow?
+outputs:
+  answer:
+    type: string
+    reference: ${generate_result.output}
+nodes:
+- name: content_safety_check
```

Comment:
Would be great if the sample could be a topic guardrail, if the answered the questions not related to prompt flow, return the default answer, otherwise, return RAG answer.  the challenge for ACS is that following LLM node also will do ACS pre-filtering of user input

Reply:
This is just a showcase of activate config. If you want to add more complex logic, you can change it after this PR is merged.